scripts/02_install_groups.sh

```diff
@@ -22,6 +22,8 @@ PKGGROUPS="$(sed '/^ *#/d; s/  *#.*//' "${PKGLISTFILE}" | sed ':a;N;$!ba; s/\n/
 PACMAN_CACHE_DIR="${CACHEDIR}/pacman_cache"
 export PACMAN_CACHE_DIR
 
+cp /etc/resolv.conf "${INSTALLDIR}/etc/"
```

Comment:
If copying `/etc/resolv.conf` is really required, I'd recommend remove it afterwards (in `09_cleanup.sh`?). To not leak build environment into the template, which breaks reproducibility (which isn't working anyway yet...).